src/accounts/nuls2.ts

```diff
@@ -22,8 +24,27 @@ export class NULS2Account extends Account {
         this.privateKey = privateKey;
     }
 
-    GetChain(): ChainType {
-        return ChainType.NULS2;
+    GetChain(): Chain {
+        return Chain.NULS2;
+    }
+
+    /**
+     * Encrypt a content using the user's public key for a NULS2 account.
+     *
+     * @param content The content to encrypt.
+     */
+    encrypt(content: Buffer): Buffer {
+        return secp256k1_encrypt(this.publicKey, content);
+    }
+
+    /**
+     * Decrypt a given content using a NULS2 account.
+     *
+     * @param encryptedContent The encrypted content to decrypt.
+     */
+    decrypt(encryptedContent: Buffer): Buffer {
+        const secret = this.privateKey;
+        return secp256k1_decrypt(secret, encryptedContent);
```

Comment:
No need to create a temp. variable here

Reply:
Fixed in [c099ae5](https://github.com/aleph-im/aleph-sdk-ts/pull/10/commits/c099ae559076917cfc9b9007f866d9dbe09bce07)